install.sh

```diff
@@ -1,4 +1,8 @@
-#!/bin/sh
+#!/bin/zsh
 
 echo "Installation script is deprecated!"
 echo "For installation instruction please visit https://github.com/nvbn/thefuck"
+echo ""
+echo "To install from sources run:"
+echo "python setup.py build && python setup.py install"
+python setup.py build && python setup.py install
```

Comment:
If a user comes to this point they know how to install things in their Python environment (or virtual environment). There's no intention of recommending anything at this point.

Also, this should not be part of the pull request.